app/components/Chat.vue.ts

```diff
@@ -25,25 +26,41 @@ export default class Chat extends Vue {
     const platform = this.userService.platform.type;
     const service = getPlatformService(platform);
     const nightMode = this.customizationService.nightMode ? 'night' : 'day';
+    const webview = this.$refs.chat;
+    const settings = this.customizationService.getSettings();
 
     service.getChatUrl(nightMode).then(chatUrl => this.chatUrl = chatUrl);
 
-    this.$refs.chat.addEventListener('new-window', e => {
+    webview.addEventListener('new-window', e => {
       const protocol = url.parse(e.url).protocol;
 
       if (protocol === 'http:' || protocol === 'https:') {
         electron.remote.shell.openExternal(e.url);
       }
     });
 
-    this.$refs.chat.addEventListener('dom-ready', () => {
-      this.$refs.chat.setZoomFactor(this.customizationService.state.chatZoomFactor);
+    webview.addEventListener('dom-ready', () => {
+      webview.setZoomFactor(settings.chatZoomFactor);
+      if (settings.enableBTTVEmotes && this.isTwitch) {
+        webview.executeJavaScript(`
+        
+        localStorage.setItem('bttv_clickTwitchEmotes', true);
+        
+        var bbtvscript1 = document.createElement('script');
+        bbtvscript1.setAttribute('src','https://cdn.betterttv.net/betterttv.js');
+        document.head.appendChild(bbtvscript1);
+        
+        var bbtvscript2 = document.createElement('script');
+        bbtvscript2.setAttribute('src','https://legacy.betterttv.net/betterttv.js');
+        document.head.appendChild(bbtvscript2);
```

Comment:
Can you indent this one level further.  I didn't see the `executeJavascript` call above.